tests/system_tests/locking_test.py

```diff
@@ -13,7 +14,9 @@ def test_starting_a_session_on_the_same_switch_waits_for_previous_session_to_fin
 
             pool = ThreadPool(processes=1)
 
-            create_session(client, "my_session")
+            result = create_session(client, "my_session")
```

Comment:
Since using we're non-flow control switches in sessions, this test is useless and probably works becuase the assert at line 26 is done to fast even for an instant call.

Probably that a little sleep()would show that this test does not wait for the other session to finish as there is not locking anymore

Testing the locking and still be pertinent... but this test should be rewritten to use something else, maybe flow control switches directly... or maybe we should just delete it